zebra/zebra_nhg.c

```diff
@@ -221,6 +221,17 @@ static int nexthop_active(afi_t afi, struct route_entry *re,
 		}
 	}
 
+	if ((top->p.family == AF_INET && top->p.prefixlen == 32
```

Comment:
in the other block we test 'top' - should that be done here too, can it be NULL?

Reply:
I don' t think it can be NULL, it comes passed as the `rn` from `rib_process()`